### PR TITLE
[FIX] Paint Data: Save and load labels

### PR DESCRIPTION
**Pull request by jerneju**
##### Issue
No extra labels are saved when class has more than two values.

##### Includes
- [X] Code changes
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Orange/widgets/data/owpaintdata.py [hunk 1]
```diff
@@ -778,6 +778,7 @@ class OWPaintData(OWWidget):
     density = Setting(7)
     #: current data array (shape=(N, 3)) as presented on the output
     data = Setting(None, schema_only=True)
+    labels = Setting(None, schema_only=True)
```
Comment (astaric) [inline Orange/widgets/data/owpaintdata.py]: Why not Setting(["C1", "C2"], schema_only=True)?

File: Orange/widgets/data/owpaintdata.py [hunk 2]
```diff
@@ -739,24 +720,23 @@ def data(self, index, role=Qt.DisplayRole):
...
                 0 <= index.row() < self.colors.number_of_colors:
             return gui.createAttributePixmap("", self.colors[index.row()])
-        else:
-            return super().data(index, role)
+        return super().data(index, role)
 
 
-def _i(name, icon_path="icons/paintdata",
-       widg_path=os.path.dirname(os.path.abspath(__file__))):
+def _ic(name, icon_path="icons/paintdata",
```
Comment (astaric) [inline Orange/widgets/data/owpaintdata.py]: _i was not a very good name and neither is _ic. what is wrong with _icon?

Comment (jerneju) [inline Orange/widgets/data/owpaintdata.py]: Fixed.

**Comment by astaric** [comment]
Works perfectly.

**Comment by jerneju** [comment]
https://sentry.io/biolab/orange3/issues/196745066/

**Pull request merged**